hack/verify-gofmt.sh

```diff
@@ -24,9 +24,9 @@ KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
 
 GO_VERSION=($(go version))
 
-if [[ -n $(echo "${GO_VERSION[2]}" | grep -E 'go1.1|go1.2|go1.3') ]]; then
+if [[ -n $(echo "${GO_VERSION[2]}" | grep -E 'go1.1|go1.2|go1.3|go1.4|go1.6') ]]; then
```

Comment:
we should fail on anything less than 1.4. possibly even anything less than 1.6 (as we're likely to not compile on anything <1.5 soon).